03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,42 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>The Daily Planet</title>
-        <link rel="stylesheet" href="css/daily-planet-style.css"/>
-    </head>
-    <body>
-        <!-- You code start here -->
-    </body>
-</html>
\ No newline at end of file
+
+<head>
+    <title>The Daily Planet</title>
+    <link rel="stylesheet" href="css/daily-planet-style.css" />
+</head>
+

```

Comment:
Great work! Everything is organized and clear. I would start getting into the habit of commenting your code. Other than that, more Houdini references please!